universe/info.owlbot/device.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const Tp = require('thingpedia');
+const Owlbot = require('owlbot-js')
+const client = Owlbot('5424c7b02c9bf605e23e67bfaed1f52341842ea0')
```

Comment:
Is this the API key? You'll want to store this somewhere else, and not put in a public repository. Put it in a file called secrets.json and don't commit that file. Send the API key to me so I can commit it encrypted.
To access it, declare the device using:
```
import config from @org.thingpedia.config.none(api_key=$?);
```
and set the key as `{"api_key": "..."}` in the secrets.json

Then you'll be able to access the key inside the device class with `this.constructor.metadata.auth.api_key`